src/editor/core/plugins/heading/menu.js

```diff
@@ -2,33 +2,30 @@
  * @link https://www.humhub.org/
  * @copyright Copyright (c) 2017 HumHub GmbH & Co. KG
  * @license https://www.humhub.com/licences
- *
  */
 
-import {DropdownSubmenu, blockTypeItem} from "../../menu/menu"
+import {blockTypeItem} from "../../menu/menu";
 
-function makeHeading(context) {
-    let r = {};
-    for (let i = 1; i <= 6; i++) {
-        r["makeHead" + i] = blockTypeItem(context.schema.nodes.heading, {
-            label: "H" + i,
-            title: context.translate("Change to heading")+" " + i + ' (' + Array(i + 1).join("#") + ')',
-            id: 'makeHeading'+i,
-            sortOrder: i,
-            attrs: {level: i}
-        })
-    }
-
-    return new DropdownSubmenu([r.makeHead1, r.makeHead2, r.makeHead3, r.makeHead4, r.makeHead5, r.makeHead6], {label: context.translate("Heading")});
+function makeHeading(context, i) {
+    return blockTypeItem(context.schema.nodes.heading, {
+        id: 'makeHeading' + i,
+        attrs: {level: i},
+        label: `Heading ${i}`,
```

Comment:
Is the menu item "Headling" is translateable?

Reply:
Added translating for the menu item "Heading".